lib/routes.dart

```diff
@@ -68,6 +69,7 @@ class Routes {
           const ViewTherapyScreen(),
       '/my-therapy-screen/my-therapy': (BuildContext ctx) =>
           const MyTherapiesScreen(),
+      'legals': (BuildContext ctx) => const LegalsPage(),
```

Comment:
You should stick to our convention. If you wan't to change it, we can discuss it if you want...
I've opened a new issue on that regard: #67